packages/javascript-sdk/src/device-client/services/index.ts

```diff
@@ -44,70 +51,91 @@ export const deviceService = ({ baseUrl, realmPath }: { baseUrl: string; realmPa
           `json/realms/${realm}/users/${userId}/devices/2fa/oath?_queryFilter=true`,
       }),
 
-      deleteOathDevice: builder.mutation<DeletedOAthDevice, DeleteOathQuery & OathDevice>({
-        query: ({ realm = realmPath, userId, uuid, ...body }) => ({
+      deleteOathDevice: builder.mutation<
+        DeletedOAthDevice,
+        RetrieveOathQuery & { device: OathDevice }
```

Comment:
I see this quite often throughout the codebase. Can we make sure all references to "oath" are either `OATH`, `Oath` or `oath`? Currently, I see `OAth` quite a bit, which I'd like to correct to `Oath`.